packages/fluentui/react-builder/src/config.tsx

```diff
@@ -0,0 +1,721 @@
+//       If you replace React.Children.map from Flex and just return `children`, no prefixing occurs.
+//       -- FIX?
+//       The builder could use some other way of relating JSON tree elements to React elements besides key.
+const keyToUUID = key => (typeof key === 'string' ? key.replace(/^\.\$/, '') : key);
+
+export const jsonTreeFindElement = (tree: JSONTreeElement, uuid: string | number): JSONTreeElement | null => {
+  const uuidFromKey = keyToUUID(uuid);
+
+  if (typeof uuidFromKey === 'undefined' || uuidFromKey === null) {
+    return null;
+  }
+
+  if (tree.uuid === uuidFromKey) return tree;
+
+  let ret = null;
+  if (Array.isArray(tree?.props?.children)) {
+    for (let i = 0; i < tree?.props?.children.length && ret === null; ++i) {
+      const e = tree?.props?.children[i];
+      if (typeof e !== 'string') {
+        ret = jsonTreeFindElement(e, uuidFromKey);
+      }
+    }
+  }
+  return ret;
+};
+
+export const jsonTreeFindParent = (tree: JSONTreeElement, uuid: string | number): JSONTreeElement | null => {
+  const uuidFromKey = keyToUUID(uuid);
+
+  if (typeof uuidFromKey === 'undefined' || uuidFromKey === null) {
+    return null;
+  }
+
+  let ret = null;
+  if (Array.isArray(tree?.props?.children)) {
+    if (tree.props.children.find(jte => typeof jte !== 'string' && jte.uuid === uuidFromKey)) {
+      return tree;
+    }
+    for (let i = 0; i < tree?.props?.children.length && ret === null; ++i) {
+      const e = tree?.props?.children[i];
+      if (typeof e !== 'string') {
+        ret = jsonTreeFindParent(e, uuidFromKey);
+      }
+    }
+  }
+  return ret;
+};
+
+export const jsonTreeDeleteElement = (tree: JSONTreeElement, uuid: string | number): JSONTreeElement => {
+  const uuidFromKey = keyToUUID(uuid);
+
+  const omitChildWithUuid = (tree: JSONTreeElement, uuid: string | number): JSONTreeElement => {
+    if (Array.isArray(tree?.props?.children)) {
+      tree.props.children = tree.props.children
+        .filter(jte => typeof jte === 'string' || jte.uuid !== uuid)
+        .map(jte => (typeof jte === 'string' ? jte : omitChildWithUuid(jte, uuid)));
+    }
+
+    return tree;
+  };
+
+  if (tree.uuid === uuidFromKey) {
+    return null;
+  }
+
+  return omitChildWithUuid(tree, uuid);
+};
+
+export const jsonTreeCloneElement = (tree: JSONTreeElement, element: any): JSONTreeElement => {
+  const result = _.transform(element, (acc, value, key) => {
+    if (_.isPlainObject(value)) {
+      acc[key] = jsonTreeCloneElement(tree, value);
+    } else if (Array.isArray(value)) {
+      acc[key] = jsonTreeCloneElement(tree, value);
+    } else {
+      acc[key] = value;
+    }
+  }) as any;
+
+  if (_.isPlainObject(result) && result.$$typeof === 'Symbol(react.element)' && result.uuid) {
+    result.uuid = getUUID();
+  }
+  return result;
+};
+
+/**
+ * Displays a knob with the ability to switch between data `types`.
+ */
+export const MultiTypeKnob: React.FC<{
```

Comment:
Should be `MultiTypeKnob` in this file?

Reply:
For now yes, reworked Knobs will be introduced in another PR.